pkg/ossm/smcp_install.go

```diff
@@ -55,29 +60,35 @@ func installDefaultSMCP22() {
 }
 
 func TestSMCPInstall(t *testing.T) {
-	defer installDefaultSMCP23()
+	NewTest(t).LegacyID("A1").Groups(Smoke, Full, InterOp, ARM).Run(func(t TestHelper) {
+		hack.DisableLogrusForThisTest(t)
+		t.Cleanup(func() {
+			installDefaultSMCP(t, meshNamespace)
+		})
+		vars := vars{
+			Name:      smcp.Name,
+			Namespace: smcp.Namespace,
+		}
 
+		t.NewSubTest("smcp_test_install_2.3").Run(func(t TestHelper) {
+			t.LogStep("Crete Namespace and Install SMCP v2.3")
+			oc.CreateNamespace(t, meshNamespace)
+			oc.ApplyString(t, meshNamespace, template.Run(t, smcpV23_template, vars))
+			oc.WaitAllPodsReady(t, meshNamespace)
+			oc.ApplyString(t, meshNamespace, smmr)
+			if env.Getenv("ROSA", "false") == "true" {
+				oc.PatchWithMerge(
+					t, meshNamespace,
+					fmt.Sprintf("smcp/%s", smcpName),
+					`{"spec":{"security":{"identity":{"type":"ThirdParty"}}}}`)
+				oc.WaitAllPodsReady(t, meshNamespace)
+			}
```

Comment:
You can move this before you call `oc.WaitSMCPReady()` in line 77. And you can remove the `oc.WaitAllPodsReady` on line 84.

Reply:
Done